_data/simple-icons.json

```diff
@@ -3575,6 +3575,11 @@
             "hex": "F7B93E",
             "source": "https://github.com/prettier/prettier-logo/tree/master/images"
         },
+        {
```

Comment:
[Looks like the icons are out of order](https://travis-ci.com/simple-icons/simple-icons/jobs/280201596#L249) I believe this one will have to come before "PrestaShop", but I would run `npm run our-lint` locally just to be sure 😉

Reply:
Darn!
Should have run the lint earlier. But yeah, I wasn't sure, where it belongs. Thanks for the nudge! 👋 :)